### PR TITLE
Gives barman back his new gear

### LINKED ISSUE #13536: beer-goggles, armor, and beanbags don't spawn in the bar backroom
Quite a problem for late-joining bartenders being without everything they need but their gun (which probably got stolen by the clown by that point)

### PR DESCRIPTION
**Pull request by Bawhoppen**
you know what this is about

fixes https://github.com/tgstation/-tg-station/issues/13536 again for technicality sake

### CODE REVIEW

**Comment by swankcookie** [comment]
Did they get removed by accident?

**Comment by Bawhoppen** [comment]
they're gone cause I reverted them when I just quick-reverted the big map fuckup I did

**Comment by swankcookie** [comment]
oh gotchya.  just making sure there was no :snowman: violation.

**Comment by Bawhoppen** [comment]
thats why I added the technical fix to it anyways

**Comment by Zilenan91** [comment]
Oh sweet you're giving him a bandolier too. I can get behind that.

**Comment by Bawhoppen** [comment]
ye this was already merged earlier but I fucked up with the map in the same PR so I'm doing this again

File: code/game/jobs/job/cargo_service.dm [hunk 1]
```diff
@@ -116,9 +116,9 @@ Bartender
 	ears = /obj/item/device/radio/headset/headset_srv
 	uniform = /obj/item/clothing/under/rank/bartender
 	suit = /obj/item/clothing/suit/armor/vest
-	backpack_contents = list(/obj/item/ammo_casing/shotgun/beanbag=4)
+	backpack_contents = list(/obj/item/weapon/storage/box/beanbag=1)
```
Comment (Fox-McCloud) [inline code/game/jobs/job/cargo_service.dm]: why not just give him a fully loaded bandolier?

Comment (Bawhoppen) [inline code/game/jobs/job/cargo_service.dm]: eh I could but box is more thematic I guess
bandolier is optional swag mode

**Comment by Bawhoppen** [comment]
this was already merged earlier today pls merge again soon

**Pull request merged**